tests/unit/modules/pillar_test.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+# Import Salt Testing libs
+from salttesting import skipIf, TestCase
+from salttesting.helpers import ensure_in_syspath
+from salttesting.mock import (
+    MagicMock,
+    patch,
+    NO_MOCK,
+    NO_MOCK_REASON
+)
+
+ensure_in_syspath('../../')
+
+# Import Salt libs
+from salt.utils.odict import OrderedDict
+from salt.modules import pillar as pillarmod
+
+
+pillar_value_1 = dict(a=1, b='very secret')
+
+
```

Comment:
Oh, actually I see you have that line above the functions needing them. Usually we just put that at the top of the file, but I think this works for now. Thoughts @s0undt3ch?